src/Resources/config/doctrine_mongodb.xml

```diff
@@ -16,9 +16,6 @@
         <service id="sonata.admin.guesser.doctrine_mongodb_list" class="Sonata\DoctrineMongoDBAdminBundle\Guesser\TypeGuesser">
             <tag name="sonata.admin.guesser.doctrine_mongodb_list"/>
         </service>
-        <service id="sonata.admin.guesser.doctrine_mongodb_list_filter" class="Sonata\DoctrineMongoDBAdminBundle\Guesser\FilterTypeGuesser">
```

Comment:
You can't remove a service, but you can create an alias and deprecate the service